examples/managed_instance_group/main.tf

```
@@ -82,65 +72,66 @@ module "mig_template" {
   source_image_project = "cos-cloud"
   source_image         = reverse(split("/", module.gce-container.source_image))[0]
   metadata             = merge(var.additional_metadata, map("gce-container-declaration", module.gce-container.metadata_value))
-  tags                 = [
+  tags = [
     "container-vm-test-mig"
   ]
-  labels               = {
+  labels = {
     "container-vm" = module.gce-container.vm_container_label
   }
 }
-
 module "mig" {
   source            = "terraform-google-modules/vm/google//modules/mig"
   version           = "~> 1.0.0"
   instance_template = module.mig_template.self_link
   region            = var.region
   hostname          = var.network
   target_size       = var.mig_instance_count
-  named_ports       = [
+  named_ports = [
     {
       name = "http",
       port = var.image_port
     }
   ]
-  network           = google_compute_network.default.self_link
-  subnetwork        = var.subnetwork
+  network    = google_compute_network.default.self_link
+  subnetwork = var.subnetwork
 }
-
 module "http-lb" {
-  source            = "../../../terraform-google-lb-http/"
-  project           = var.project_id
-  name              = "${var.mig_name}-lb"
+  source  = "github.com/terraform-google-modules/terraform-google-lb-http"
```

This should be sourced from the Terraform Registry with a version.

```suggestion
  source  = "GoogleCloudPlatform/lb-http/google"
  version = "~> 1.0"
 
```